rllib/agents/dqn/dqn.py

```diff
@@ -64,19 +65,37 @@
         # N-step Q learning
         "n_step": 1,
 
-        # === Prioritized replay buffer ===
-        # If True prioritized replay buffer will be used.
+        # === Replay buffer ===
+        # Size of the replay buffer. Note that if async_updates is set, then
+        # each worker will have a replay buffer of this size.
+        "buffer_size": DEPRECATED_VALUE,
+        # For now we don't use the new ReplayBuffer API here
+        "_replay_buffer_api": False,
```

Comment:
new flag? why isn't this flag part of replay_buffer_config?

Reply:
Yes, it enables the api. Thanks, I moved it and changed the name to _enable_replay_buffer_api.